generators/app/index.js

```diff
@@ -48,6 +48,7 @@ module.exports = yeoman.Base.extend({
   },
 
   install: function() {
-    this.installDependencies();
+    this.npmInstall([], {depth: 0});
```

Comment:
It would be nice but I'm not sure how we could. It isn't really necessary unless someone is testing the previous variant.